stac_fastapi/elasticsearch/stac_fastapi/elasticsearch/core.py

```diff
@@ -284,7 +313,7 @@ async def post_search(
 
         if hasattr(search_request, "filter"):
             cql2_filter = getattr(search_request, "filter", None)
-            if filter_lang in [None, FilterLang.cql2_json]:
+            if filter_lang in [None, FilterLang.cql2_json, FilterLang.cql_json]:
```

Comment:
Just double-checking that `cql_json` (not `cql2_json`) is valid to pass to `apply_cql2_filter`?